src/sec_edgar_bulker/core/__init__.py

```
@@ -0,0 +1,45 @@
+    'DownloadSettings',
+    'LoggingSettings',
+    'DirectorySettings',
+    'FilteringSettings',
+    'ConfigError',
+    'DownloadModeError',
+    'InvalidProxyFormatError',
+    'ProxyFileError',
+    'setup_logging',
+    'validate_config',
+    'generate_batch_id'
+] 
```

Ruff (0.8.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)